api-server/migrations/2021-05-27-195835_expand-db/up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE "sensors" (
+    id UUID NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4 (),
+    alias VARCHAR NOT NULL,
+    label VARCHAR NOT NULL,
+    type_id UUID NOT NULL DEFAULT uuid_generate_v4 (),
+    station_id UUID NOT NULL DEFAULT uuid_generate_v4 (),
+    /*created_at TIMESTAMP NOT NULL DEFAULT NOW (),
+    updated_at TIMESTAMP NOT NULL DEFAULT NOW ()*/
+    created_at VARCHAR NOT NULL,
+    updated_at VARCHAR NOT NULL
+);
+
+CREATE TABLE "measurements" (
+    id UUID NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4 (),
+    value integer NOT NULL,
```

Comment:
Using value **NUMERIC** NOT NULL will give us the desired decimal precision for any arbitrary measurement type. According to [this chart](https://kotiri.com/2018/01/31/postgresql-diesel-rust-types.html) you would need to change the MeasurementModel struct value from an i32 to a BigDecimal (as well as importing bigdecimal in Cargo.toml). Let me know if it gives you any grief though.